README.md

```diff
@@ -7,13 +7,33 @@ Undo/Redo feature for [Editor.js](https://editorjs.io).
 ![](assets/demo.gif)
 
 ## Installation
-
 ### Install via NPM
 
+### 2.x.x (beta)
+
+This new version 2.0.0-rc.0 introduces breaking changes. Remember that this is a beta version.
+
+Please, report bugs or behavior issues :).
+
+#### What's new?
+* Overall plugin performance is improved, specially in large documents. 
```

Comment:
@MarioRodriguezS a typo over here, it should be **especially**

Reply:
@MarioRodriguezS IMO, "especially **with** large documents" may work better in this case. What do you think?

Reply:
@juanhurtado10 sure!, thanks.